apps/web/components/dashboard/bookmarks/EditorCard.tsx

```diff
@@ -150,6 +176,43 @@ export default function EditorCard({ className }: { className?: string }) {
               : "Press ⌘ + Enter to Save"
             : "Save"}
         </ActionButton>
+
+        {multiUrlImportState && (
+          <MultipleChoiceDialog
```

Comment:
The user can still dismiss the dialog manually (by clicking outside the dialog box for example), and in that case, you won't be able to open the dialog again.

You'll still need to subscribe to `onOpenChange` and reset the state if it gets closed.
```suggestion
          <MultipleChoiceDialog
            onOpenChange={(open) => {
              if (!open) {
                setMultiUrlImportState(null);
              }
            }}
          
```